src/test/java/net/snowflake/client/core/arrow/BigIntToTimestampNTZConverterTest.java

```diff
@@ -62,7 +63,16 @@ public void testHonorClientTZForTimestampNTZEnabled() throws SFException {
     testTimestampNTZ();
   }
 
+  @Test
```

Comment:
Add a test for case where timezone could be null (in this case, default timezone is used). Add test to check that the default timezone is used.